hr_timesheet_auto_creation/models/hr_timesheet_sheet.py

```diff
@@ -35,14 +35,20 @@ def create_employee_timesheet(self):
             lambda x: x.employee_id.id, exists_timesheet_records
         )
         employee_ids = list(set(employee_ids) - set(ignore_employee_ids))
+        sudo_self = self.sudo()
         for employee_id in employee_ids:
-            self.sudo().create(
+            tss = sudo_self.create(
```

Comment:
I would probably rename this to `sheet` or some such as a more descriptive variable.

Reply:
indeed! `tss` means “timesheet sheet”, but it’s a little cryptic.